com.unity.shadergraph/Editor/Generation/Descriptors/PragmaDescriptor.cs

```diff
@@ -32,5 +32,6 @@ static string GetPlatformList(Platform[] platforms)
         public static PragmaDescriptor DOTSInstancing => new PragmaDescriptor { value = "multi_compile _ DOTS_INSTANCING_ON" };
         public static PragmaDescriptor MultiCompileFog => new PragmaDescriptor { value = "multi_compile_fog" };
         public static PragmaDescriptor EditorSyncCompilation => new PragmaDescriptor { value = "editor_sync_compilation" };
+        public static PragmaDescriptor Debug() => new PragmaDescriptor { value = "enable_d3d11_debug_symbols" };
```

Comment:
Is this hooked up to anything?

Reply:
Whenever I need to use RenderDoc for debugging, I add that to the shader passes, so I thought it was convenient to have it available easily. Is there already a quick way of turning on debugging?